samples/dashboards/demos/climate/demo.css

```diff
@@ -209,31 +201,79 @@
  *
  * */
 
-.hd-component-kpi,
-.hd-component-kpi-value-wrap {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-    justify-content: space-evenly;
+#kpi-layout .hd-component {
+    background-color: var(--color-background);
+    color: var(--color-text);
+    cursor: pointer;
 }
 
-.hd-component-kpi > *,
-.hd-component-kpi-value-wrap > * {
+#kpi-layout .hd-component-kpi-content {
+    background: none;
+}
+
+#kpi-layout .hd-component-kpi-chart-container {
     align-self: center;
-    flex: none;
     margin: auto;
 }
 
-.hd-component-kpi {
-    text-align: center;
+#kpi-layout .hd-component-kpi-chart-container text {
+    fill: var(--color-text);
 }
 
-.hd-component-kpi-content {
-    background: none;
+#kpi-layout .hd-component-kpi-chart-container .highcharts-background {
+    fill: none;
+    stroke: none;
 }
 
-.hd-component-kpi-value-wrap {
-    height: 100%;
+#kpi-layout .hd-component-kpi-chart-container .highcharts-label {
+    font-size: 32px;
+}
+
+#kpi-layout .hd-component-kpi-chart-container .highcharts-pane {
+    fill: var(--color-pane);
+    fill-opacity: 1;
+    stroke: var(--color-pane-line);
+}
+
+#kpi-layout .hd-component-kpi-chart-container .highcharts-series .highcharts-color-0 {
+    fill: var(--color-gauge);
+    stroke: none;
+}
+
+#kpi-layout .hd-component-kpi-chart-container .highcharts-title {
+    font-size: 13px;
```

Comment:
I feel like the `font-size` might be slightly bigger. I can't see that in figma.

Reply:
Probably, but we are out of space anyway.